events/elections/2018/README.md

```diff
@@ -2,63 +2,91 @@
 
-## Voting
+## Background
 
-This election will  shape the future of Kubernetes as a project and community. You will be voting on three (3) seats up for election.
-Every elected community member will serve a two year term.
+This election will shape the future of Kubernetes as a community and project.
+While SIGs and WGs help shape the technical direction of the project, the
+[Steering Committee Charter] covers the health of the project and community
+as a whole. Some examples of responsibilities to consider as you are deciding
+whether to run or who to vote for:
 
-The candidates you are voting on will be meeting regularly to strategically grow the project with contributors in mind. Most of the technical decisions, architecture planning, and the like come out of SIGs and other working groups.
-Some examples of responsibilities to consider as you are voting:  
--  Define, evolve, and defend the vision, values, mission, and scope of the project - to establish and maintain the soul of Kubernetes.
--  Charter and refine policy for defining new community groups (including Special Interest Groups, Working Groups, and Committees), and establish transparency and accountability policies for such groups.  
--  Define, evolve, and defend a Code of Conduct, which must include a neutral, unbiased process for resolving conflict
+- Define, evolve, and defend the vision, values, mission, and scope of the
+  project.
+- Define and evolve project governance structures and policies, including how
+  contributors become committers/maintainers, approvers, reviewers, members,
+  etc.
+- Charter and refine policy for defining new community groups (including
+  Special Interest Groups, Working Groups, and Committees), and establish
+  transparency and accountability policies for such groups
 
-Full scope of responsibilities, goals, and/or future election timelines, see [steering committee charter](https://github.com/kubernetes/steering/blob/master/charter.md).
+For more context, please see the [current steering committee backlog] or a
+previous [governance meeting video] which led to this whole process.
 
-For more context, please see the [current issues](https://github.com/kubernetes/steering/blob/master/backlog.md) that need to be resolved or a previous governance [meeting video](https://www.youtube.com/watch?v=ltRKXLl0RaE&list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ&index=23) which led to this whole process.
+## Eligibility
 
-## Process
+Please refer to the [Steering Committee Election Charter] for:
 
-Elections will be held using time-limited [Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) ranking on [CIVS](http://civs.cs.cornell.edu/) using the [Schulze method](https://en.wikipedia.org/wiki/Schulze_method). 
-The top vote getters, modulo the corporate diversity requirement of no more than 1/3 of the committee from a single company, will be elected to the respective positions.
+- [elegibility for candidacy]
+- [eligibility for voting]
```

Comment:
These links will only work once https://github.com/kubernetes/steering/pull/68 is merged